cmd/ocm-backplane/login/login.go

```diff
@@ -79,6 +87,13 @@ func runLogin(cmd *cobra.Command, argv []string) (err error) {
 		clusterKey = clusterInfo.ClusterID
 	}
 
+	// Init Proxy url
+	err = utils.DefaultOCMInterface.SetBackplaneProxyURL(args.proxyURL)
```

Comment:
We should only set if proxyURL is not empty

Reply:
yes . always check proxy URL is not empty before setting it to the env vars 
 https://github.com/openshift/backplane-cli/blob/67e50c71be30aad3c328a63cb3972ea2438bb9fc/pkg/utils/ocmWrapper.go#L270

Reply:
Done